BentShapeRebar.py

```diff
@@ -274,7 +274,7 @@ def editBentShapeRebar(Rebar, f_cover, b_cover, l_cover, r_cover, diameter, t_co
         sketch.Support = [(structure, facename)]
     # Check if sketch support is empty.
     if not sketch.Support:
-        showWarning("You have checked remove external geometry of base sketchs when needed.\nTo unchecked Edit->Preferences->Arch.")
+        showWarning("You have checked: 'remove external geometry of base sketches when needed'.\nTo uncheck: Edit->Preferences->Arch.")
```

Comment:
Does this look correct?  
Should it be capitalized like: `'Remove...'` 
Also, should it be `'Remove external geometry of base sketches when needed.'` (with the period in the quotes?

Reply:
Yes, it should print 'Remove external geometry of base sketches when needed.' in the FreeCAD console with quotes. Also, please capitalize `Remove`.

Reply:
See 4c993ae